config-reloader/datasource/fake.go

```diff
@@ -28,6 +28,13 @@ type fakeDatasource struct {
 	hashes map[string]string
 }
 
+// NewFakeDatasource returns a predefined set of namespaces + configs
```

Comment:
Looks like change this file can also be avoided, I don't see any change apart from moving code from down to up. It would be good to avoid such changes.